### PR TITLE
Add PHP unit test plumbing

### PR DESCRIPTION
**Pull request by psealock**
Fixes https://github.com/woocommerce/navigation/issues/16

Adds unit testing for PHP.

I copied much of the files from wc-admin and removed some bootstrap files that aren't necessary. I also added one sample test that we can remove once an actual test is added.

### Test

1. Setup unit tests `bin/install-wp-tests.sh wc_admin_test root 'root' localhost latest`. If this is your local dev environment with unit test already setup for wc-admin, you can skip this step.
2. `vendor/bin/phpunit -c phpunit.xml.dist`
3. See the example unit test pass.

### CODE REVIEW

File: tests/bootstrap.php [hunk 1]
```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * WooCommerce Admin Unit Tests Bootstrap
```
Comment (joshuatf) [inline tests/bootstrap.php:3]:
```suggestion
 * WooCommerce Navigation Unit Tests Bootstrap
```

File: tests/bootstrap.php [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+ *
+ * @package WooCommerce Admin Tests
```
Comment (joshuatf) [inline tests/bootstrap.php:5]:
```suggestion
 * @package WooCommerce Navigation Tests
```

File: tests/bootstrap.php [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+		// load the WP testing environment.
+		require_once $this->wp_tests_dir . '/includes/bootstrap.php';
+
+		// load WC testing framework.
+		$this->includes();
+	}
+
+	/**
+	 * Load WooCommerce Admin.
```
Comment (joshuatf) [inline tests/bootstrap.php:70]:
```suggestion
	 * Load WooCommerce Navigation.
```

File: phpunit.xml.dist [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+	bootstrap="tests/bootstrap.php"
+	backupGlobals="false"
+	colors="true"
+	convertErrorsToExceptions="true"
+	convertNoticesToExceptions="true"
+	convertWarningsToExceptions="true"
+	verbose="true"
+	>
+	<testsuites>
+		<testsuite name="WooCommerce Admin Test Suite">
```
Comment (joshuatf) [inline phpunit.xml.dist:12]:
```suggestion
		<testsuite name="WooCommerce Navigation Test Suite">
```

**Comment by psealock** [comment]
>trying to run ./bin/install-wp-tests.sh or ./bin/phpunit.sh both resulted in permission denied errors

is it a chmod type of permission error?

**Comment by psealock** [comment]
I ran the command and committed it in https://github.com/woocommerce/navigation/pull/19/commits/410905fdd3015011c59434f63f7678903ed17d10. Hopefully that works.

**Pull request merged**